IPRotate.py

```diff
@@ -542,7 +556,7 @@ def getUiComponent(self):
 		for region in AVAIL_REGIONS:
 			cur_region = region.replace('-','_')
 			cur_region = cur_region+'_status'
-			setattr(self,cur_region,JCheckBox(region,True))
+			setattr(self,cur_region,JCheckBox(region,False))

```

Comment:
Change back to True since region selection is automated now if you just leave them all checked.